<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>915</width>
    <height>446</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <widget class="QWidget" name="form">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>730</width>
      <height>400</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

QGroupBox {
	/*
	background-image: url(D:/CODE/ManageStudent/UI/BG/BG.jpg);
    background-repeat: no-repeat;
    background-position: center;
    background-origin: content;
    background-clip: border;
	*/
	background: qlineargradient(
        x1: 0, y1: 0, 
        x2: 0, y2: 1, 
        stop: 0 #a2d2ff,    /* màu trên cùng */
        stop: 1 #cdb4db    /* màu dưới cùng */
    );
	
	border: 1px solid rgb(129, 129, 129);
	
	border-radius:10px;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QLineEdit" name="password">
     <property name="geometry">
      <rect>
       <x>165</x>
       <y>215</y>
       <width>400</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#password {
    background-color: rgba(255, 255, 255, 255); /* màu trắng, không trong suốt */
    color: black;
    border: 2px solid rgb(129, 129, 129);
	padding: 4px 5px;
	border-radius:6px;

	color: black;  /* màu placeholder */
    font-style: italic;
}

#password::hover {
    background-color:rgb(231, 231, 231) /* màu trắng, không trong suốt */

}
</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="placeholderText">
      <string>password</string>
     </property>
    </widget>
    <widget class="QPushButton" name="loginBtn">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>300</y>
       <width>211</width>
       <height>45</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">#loginBtn {
	font: 81 14pt &quot;Rockwell Extra Bold&quot;;
	background-color: rgba(65, 90, 112, 255);
	border-radius: 10px;
	color: rgb(240, 240, 240);
}

#loginBtn::hover{
	font-size:16pt;
	background-color: rgb(9, 145, 0);
	border-radius: 10px;
	color: #fff;
}</string>
     </property>
     <property name="text">
      <string>login</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="userName">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>165</x>
       <y>140</y>
       <width>400</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#userName {
    background-color: rgba(255, 255, 255, 255); /* màu trắng, không trong suốt */
    color: black;
    border: 2px solid rgb(129, 129, 129);
	padding: 4px 5px;
	border-radius:6px;

	color: black;  /* màu placeholder */
    font-style: italic;
}

#userName::hover {
    background-color:rgb(231, 231, 231) /* màu trắng, không trong suốt */

}

</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="placeholderText">
      <string>user name</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="closeBtn">
     <property name="geometry">
      <rect>
       <x>685</x>
       <y>5</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">#closeBtn{
	border-radius:15px;
	border:none;
	background-color:rgb(212, 212, 212)
}

#closeBtn::hover{
	background-color:rgb(255, 62, 65);
}


</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../icons/close-outline.svg</normaloff>../icons/close-outline.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>272</x>
       <y>40</y>
       <width>186</width>
       <height>58</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Rockwell Extra Bold</family>
       <pointsize>30</pointsize>
       <weight>10</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">#label{
	/* color: rgb(58, 58, 58) */
	color: #fff;
	
	font: 81 30pt &quot;Rockwell Extra Bold&quot;;
}
</string>
     </property>
     <property name="text">
      <string>LOGIN</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="hiddenBtn">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>5</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#hiddenBtn{
	border-radius:15px;
	border:none;
	background-color:rgb(212, 212, 212)
}

#hiddenBtn::hover{
	background-color:rgb(138, 138, 138);
}


</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../icons/remove-outline.svg</normaloff>../icons/remove-outline.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
